src/service/real-time-config/real-time-config-impl.js

```diff
@@ -239,7 +241,8 @@ export class RealTimeConfigManager {
 
   /**
    * Goes through the RTC config, and for any URL that we should not callout
-   * as per the current consent state, deletes it from the RTC config.
+   * as per the current consent state and the `always-serve-npa` signal
+   * presence, deletes it from the RTC config.
```

Comment:
I would not expect always-serve-npa to prevent RTC callouts?  RTC is general-purpose, and the publisher could be using it for purposes that are consistent with NPA.

In most cases, if they don't want the callout to happen, they could not include the RTC config?  The main problem is that your always-serve-npa does allow country-group-specific handling, but I don't think there is a way to only send RTC callouts for some countries?

@mlb7687 thoughts?

Reply:
Synced offline: Makes sense to decouple the blocking RTC feature from serving NPA as serving ads and making RTC callouts are distinct.